### DISCUSSION TITLE
Does anyone else have issues with getting IPv6 to work in an Ubuntu VM?

### DISCUSSION THREAD

**Discussion by flavorgold1** [utmapp/UTM#5186] [UTM Help (macOS)]
IPv4 works just fine.

But, I can't seem to get IPv6 working.

I can't ping an IPv6 address, for example: `ping 2606:4700:4700::1111`.

What do I need to change here?

**Comment by lpar** [comment]
Had the same problem with RHEL 9. Switching the network interface to "Emulated VLAN" made it work, at least for outgoing traffic, but you end up with a site-local `fec0` address, which is deprecated from the IPv6 standards. The emulated VLAN option also seems to break IPv4. I tried adding a second interface, bridged, for IPv4, but then IPv6 stopped working again. Then I deleted the second interface and went back to just a bridged interface on en0, and suddenly everything worked. My guess is that at some point the defaults for network interfaces got changed to make it work, but that you need to remove and re-add the interface?
